README.md

```diff
@@ -85,6 +85,22 @@ Using a local Ollama instance is necessary if you're running RAGapp on macOS, as
 
 To enable Docker access to NVIDIA GPUs on Linux, [install the NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html).
 
+#### Manage RAGapp via Manager UI:
+You can set up Traefik reverse proxy and handle your apps right from the manager UI.
+
+Once you deploy RAGapp using the default `docker-compose.yml`, just use the `docker-compose.manager.yml` to get Traefik and Manager UI up and running.
```

Comment:
TBD: we could add the manager UI directly to `docker-compose.yml`, but I think we will end up with an `deployments` folder where we will have different deployment scenarios

Reply:
> we will end up with an deployments folder where we will have different deployment scenarios

Yes, that makes sense to me. For now, i think we can have two deployment scenarios: Just start single RAGapp without manager (current docker-compose) and new docker-compose with manager included.